### PR TITLE
development -> main

### LINKED ISSUE #274: bug: input should resize on "Max" input
https://user-images.githubusercontent.com/36751902/163606556-d05dda21-be21-4e76-b688-5a3dcc3d4273.mov

Otherwise, resizes fine

https://user-images.githubusercontent.com/36751902/163606642-bc81fd17-4479-4e1f-b374-b9e16a87d3aa.mov

### LINKED ISSUE #275: bug: no signer when processing
Screen Shot 2022-04-13 at 2 45 42 PM

### LINKED ISSUE #280: bug: blacklist banner doesn't update with Failed Homes
Failed networks get filtered out accurately, but a banner should appear to alert users of the outage

### PR DESCRIPTION
**Pull request by ErinHales**
Closes #280 
Closes #275 
Closes #274 

Changes:
- Update SDK
- "Complete Transfer" animation
- Max Button
- yarn instead of npm

Outstanding work:
 - [x] Rebase off of main
 - [x] #274 
 - [x] #275

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,27 +6,30 @@ This change log is manually updated at the moment.
 Unreleased
 --------------------------------
 - WalletConnect support
+- SDK upgrade
```
Comment (Imti) [inline CHANGELOG.md:9]: This + fix failedhomes banner display should be moved to release

File: src/App.vue [hunk 2]
```diff
@@ -8,7 +8,7 @@
         title="Under maintenance, temporarily unavailable:"
       >
         <span
-          v-for="domain in failedHomes"
+          v-for="(domain, i) in failedHomes"
```
Comment (Imti) [inline src/App.vue:11]: Only need domain here

File: src/store/modules/connext.ts [hunk 3]
```diff
@@ -136,7 +136,7 @@ const actions = <ActionTree<ConnextState, RootState>>{
     commit(types.SET_QUOTE, undefined)
   },
 
-  async getTransferQuote({ rootState, commit, dispatch }) {
+  async getTransferQuote({ commit, dispatch }) {
```
Comment (Imti) [inline src/store/modules/connext.ts:139]: can remove rootState here

File: src/store/modules/sdk.ts [hunk 4]
```diff
@@ -77,24 +50,26 @@ const mutations = <MutationTree<SDKState>>{
...
-      throw new Error("Couldn't setup Nomad")
+    await import('@nomad-xyz/sdk-bridge').then((sdk) => {
+      nomadSDK = sdk
+      nomad = new sdk.BridgeContext(environment)
+    })
+    Object.values(networks).forEach(({ name, rpcUrl }) => {
+      nomad.registerRpcProvider(name, rpcUrl)
+    })
+    if (environment === 'production') {
+      nomad.registerRpcProvider('xdai', process.env.VUE_APP_XDAI_RPC!)
```
Comment (Imti) [inline src/store/modules/sdk.ts:68]: Do we need this xdai thing specifically?

File: src/views/Transfer/Input/Input.amount.vue [hunk 5]
```diff
@@ -182,17 +195,25 @@ export default defineComponent({
...
+    },
   },
   watch: {
     token(newToken) {
       // update the usd amt if the token if changed outside of this component
       this.updateAmtInUSD(newToken.coinGeckoId)
     },
     async amt(newAmt) {
-      this.v$.amt.$touch()
+      // this.v$.amt.$touch()
```
Comment (Imti) [inline src/views/Transfer/Input/Input.amount.vue:216]: Not needed anymore?

**Review by Imti — APPROVED** [review]
🎃

File: src/App.vue (hunk 2 above)
Comment (ErinHales) [reply src/App.vue:11]: Actually, need index down below

File: src/store/modules/sdk.ts (hunk 4 above)
Comment (ErinHales) [reply src/store/modules/sdk.ts:68]: Yes, until we fix this thing in the GUI.  Currently, we need to register an RPC for every network in the config, regardless of whether we're actually using it
Issue: https://github.com/nomad-xyz/monorepo/issues/136

**Pull request merged**